core/src/main/java/org/elasticsearch/bootstrap/JNANatives.java

```diff
@@ -170,4 +172,16 @@ static void addConsoleCtrlHandler(ConsoleCtrlHandler handler) {
         }
     }
 
+    static void trySeccomp() {
+        if (Constants.LINUX && "amd64".equals(Constants.OS_ARCH)) {
+            try {
+                Seccomp.installFilter();
+                LOCAL_SECCOMP = true;
+            } catch (Exception e) {
+                // this is likely to happen unless the kernel is newish, its a best effort at the moment
+                // so unfortunately, we hide the stacktrace for now...
+                logger.warn("unable to install seccomp filter: " + e.getMessage());
```

Comment:
What do you think about `DEBUG` or `TRACE` log with the exception and if DEBUG/TRACE isn't enabled we just log what is there now? I remember a stack trace from one of these helped in the past